app/views/comments/_single_comment.html.erb

```diff
@@ -59,8 +59,10 @@
         <% end %>
       <% end %><!-- end caching -->
 
+      <% if logged_in_as_admin? %>
+        <p class="comment ip"><%= ts("IP Address: %{ip_address}", ip_address: single_comment.ip_address.blank? ? "No address recorded" : single_comment.ip_address) %></p>
```

Comment:
The none working code is what I pushed, however I am not sure if I need to do the loading all the skins in to the database and re cache all the skins or not.